examples_utils/notebook_logging/gc_logger.py

```diff
@@ -31,8 +31,7 @@ class GCLogger(object):
 
     _PROC_LIST = []
 
-    _BUCKET_NAME = "paperspace-uploading-test-bucket"
-    _FIREHOSE_STREAM_NAME = "paperspacenotebook_development"
+    _FIREHOSE_STREAM_NAME = "paperspacenotebook_production"
```

Comment:
Is this something we might want to configure based on environment? I can imagine that we might want to direct logging during tests to the test stream?

Maybe do a, what do you think?
```suggestion
    _FIREHOSE_STREAM_NAME = os.getenv("FIREHOSE_STREAM_NAME", "paperspacenotebook_production")
```